qchem/pennylane_qchem/qchem.py

```diff
@@ -0,0 +1,532 @@
+# Copyright 2018-2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The PennyLane quantum chemistry package. Supports OpenFermion, Psi4,
+and PySCF for quantum chemistry calculations using PennyLane."""
```

Comment:
Nice @josh146, adding the license and the module description was a pending task.